src/server-extension/resolvers/VideosResolver/index.ts

```diff
@@ -107,7 +107,7 @@ export class VideosResolver {
     idsQuerySql = extendClause(
       idsQuerySql,
       'FROM',
-      `LEFT JOIN "processor"."video_view_event" ` +
+      `INNER JOIN "processor"."video_view_event" ` +
```

Comment:
For now, I set this to inner join, but I'll ask the atlas team, what approach should we take. So please wait with merging until we decide

Reply:
We decided to keep the inner join here in the end